### PR TITLE
fixing radio button centering

### PR DESCRIPTION
**Pull request by AAloshine-scottlogic**
## Description
added the same styling of the toggle switches to the radio buttons for focusing and hovering, recentered the selected radio button dot

## Screenshots
before
image

after
image
image

note: maybe text align to the left so the overflow text on the next line isn't as close to the highlight? or i could increase the margin from 0.5 to 0.6rems to give the radio button a little more space?

## Checklist

Have you done the following?

- [x] Linked the relevant Issue
https://github.com/ScottLogic/prompt-injection/issues/754
- [ ] Added tests
- [ ] Ensured the workflow steps are passing

### CODE REVIEW

**Comment by chriswilty** [comment]
@AAloshine-scottlogic I'd say we put the radiobuttons and labels into some sort of container - flex or grid - so that when the text wraps, it looks like this:

radiobuttons

File: frontend/src/components/DefenceBox/DefenceConfigurationRadioButton.css
Comment (pmarsh-scottlogic) [inline frontend/src/components/DefenceBox/DefenceConfigurationRadioButton.css:35]: This change has caused the little "pupil" of the bottom two radio buttons to be off-centre. What was the intention of the change?

image

**Review by pmarsh-scottlogic — CHANGES_REQUESTED** [review]
Just one wee problem from me. Otherwise, solid.

File: frontend/src/components/DefenceBox/DefenceConfigurationRadioButton.css
Comment (AAloshine-scottlogic) [reply frontend/src/components/DefenceBox/DefenceConfigurationRadioButton.css:35]: the original had the button off centered (if you look at the before picture)

Comment (pmarsh-scottlogic) [reply frontend/src/components/DefenceBox/DefenceConfigurationRadioButton.css:35]: weird. For me it's on centre on my machine (big 1080p monitor, full screen, no zoom), and then off-centre after the change

Comment (AAloshine-scottlogic) [reply frontend/src/components/DefenceBox/DefenceConfigurationRadioButton.css:35]: well its not in the scope of this ticket so i'll just revert it

File: frontend/src/components/DefenceBox/DefenceConfigurationRadioButton.css
Comment (chriswilty) [inline frontend/src/components/DefenceBox/DefenceConfigurationRadioButton.css:5]: Remove this, seems unnecessary. We already have `gap: 0.5rem` on the parent element, so if you want more vertical space between radios, bump that up a little.

Also, I'd prefer `gap: 0.75rem` on this class, instead of margin-left on `.defence-radio-button span` below.

File: frontend/src/components/DefenceBox/DefenceConfigurationRadioButton.css
Comment (chriswilty) [inline frontend/src/components/DefenceBox/DefenceConfigurationRadioButton.css:4]: Remove the `display: inline-block` here.

**Review by chriswilty — CHANGES_REQUESTED** [review]
Looks good, very minor change suggestions only.

What Peter said about the off-centred dot, is what I was hinting at with my "check at various zoom levels" because for whatever reason, this radiobutton styling is not consistent across different browsers or zoom levels.

I am minded to make it all a bit simpler anyway... But that's for another day!

File: frontend/src/components/DefenceBox/DefenceConfigurationRadioButton.css
Comment (chriswilty) [reply frontend/src/components/DefenceBox/DefenceConfigurationRadioButton.css]: I think we can solve this with a simpler css structure, but that's for another day. I've seen perfectly good radiobuttons with custom styling that only have a `::before` pseudo-element, not both before and after, which is what seems to be throwing alignment off here.

**Comment by chriswilty** [comment]
@pmarsh-scottlogic Just waiting on a re-review from you, then this one can go in.

**Review by pmarsh-scottlogic — APPROVED** [review]
fab

**Pull request merged**
